core/classes/Core/Util.php

```diff
@@ -283,6 +283,22 @@ public static function replaceAnchorsWithText(string $data): string {
         return URL::replaceAnchorsWithText($data);
     }
 
+    private static function getSettingsCache(?string $module): ?array {
+        $cache_name = $module !== null ? $module : 'core';
```

Comment:
I take it there's no issue with this being lower case for `core` whereas the official module name is `Core`

Reply:
there's nothing that enforces modules to use their settings, nothing prevents the store module from calling `setSetting('something', 'value', $module='suggestions')`, for example. I assumed lower case, but case matching module names might make more sense.